test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -145,6 +145,10 @@ class SkipTest(Exception):
     'num_moms': 1,
     'num_comms': 1,
     'num_clients': 1,
+    'min_mom_ram' : 500,
+    'min_mom_disk' : 500,
+    'min_server_ram' : 500,
```

Comment:
Don't you think 500Mb is less? pbs_benchpress will eat up ~300MB of RAM when you run performance tests... I think the default should be at least 1Gb